lib/src/draw_config_context.dart

```diff
@@ -114,28 +122,27 @@ class DRAWConfigContext {
   String _userAgent;
   String _username;
 
-  Uri get accessToken => Uri.parse(_accessToken);
-  Uri get authorizeUri => Uri.parse(_authorizeUri);
-  Uri get redditUrl => Uri.parse(_redditUrl);
-  Uri get redirectUri => Uri.parse(_redirectUri);
-  Uri get revokeToken => Uri.parse(_revokeToken);
-
+  String get accessToken => _accessToken;
+  String get authorizeUri => _authorizeUri;
```

Comment:
Change these from Uri to Url so we don't get these fields confused with actual `Uri` instances.